lib/flame/dispatcher.rb

```diff
@@ -29,6 +29,7 @@ def initialize(app_class, env)
 			@env = env
 			@request = Flame::Dispatcher::Request.new(env)
 			@response = Flame::Dispatcher::Response.new
+			@endpoint = @app_class.router.routes.endpoint(*request.path.parts)
```

Comment:
Maybe lazy-loading via getter-method? And more obvious name, like `available_endpoint`, for example.

Reply:
as you wish!

Reply:
Just for you understanding: there is no point in wasting resources to find a route if static requested.